pkg/addon/addon.go

```diff
@@ -990,20 +992,53 @@ func (h *Installer) getAddonMeta() (map[string]SourceMeta, error) {
 // installDependency checks if addon's dependency and install it
 func (h *Installer) installDependency(addon *InstallPackage) error {
 	var dependencies []string
+	var addonClusters = getClusters(h.args)
+	// addon must be installed in local now (the render logic dependency addon crd in local)
+	addonClusters = append(addonClusters, multicluster.ClusterLocalName)
```

Comment:
I think we needn't add this logic. An addon should can be installed in non-local cluster, if it doesn't contain any CRD.

Reply:
yes, but we must ensure the addon which contains crd can be installed in non-local cluster. this logic is needed by the addon which contains crd.

Reply:
This can be judged by user. We needn't add any hack code.

Reply:
you means, if user run `vela addon enable fluxcd --clusters={devops}` first(thus only devops cluster has the fluxcd). then run `vela addon enable rollout --clusters={devops}`,, then the devops and local cluster both install rollout, but bacause the local cluster does not have fluxcd, the rollout in local will not be installed successfully. In this case, should let the user install fluxcd in the local cluster manually, rather than in the hard-coded automatic installation the fluxcd to the user?

Reply:
Yes,  In fact, the fluxcd addon cannot work well currently, If user only install it in `devops` cluster not local cluster. But this may be fixed in the future, we needn't add this constraint in addon's logic.